src/postgraphile/cli.ts

```diff
@@ -82,6 +82,7 @@ program
   .option('-p, --port <number>', 'the port to be used. Defaults to 5000', parseFloat)
   .option('-m, --max-pool-size <number>', 'the maximum number of clients to keep in the Postgres pool. defaults to 10', parseFloat)
   .option('-r, --default-role <string>', 'the default Postgres role to use when a request is made. supercedes the role used to connect to the database')
+  .option('--pre-request <string>', 'the Postgres identifier for a function that will be called before each request')
```

Comment:
Let's rename it to `--transaction-validation-function` as that's effectively its purpose.